src/marlin_stubs/M865.cpp

```diff
@@ -11,6 +11,7 @@
  *
  *#### Parameters
  * - `I<ix>` - Configure parameters of a Custom filament currently loaded to the specified tool (indexed from 0)
+ * - `I<i>`  - Show the parameters of the filament currently loaded to the specified tool (indexed from 0)
```

Comment:
- The rest is using the "Configure parameters" text. I'd suggest either updating all texts, or keeping it unified.
- The rest is using "ix", not "i"
- Also, you've duplicated the documentation for I.